tests/sample_data/diag_scripts/__init__.py

```diff
@@ -0,0 +1,116 @@
+"""Test diagnostics scripts with sample data."""
+
+from __future__ import annotations
+
+from collections.abc import Iterable
+from pathlib import Path
+
+import yaml
+from imagehash import hex_to_hash, phash
+from PIL import Image
+
+SAMPLE_DATA_DIR = Path(__file__).resolve().parent / "data"
+with (SAMPLE_DATA_DIR / "metadata.yml").open("r", encoding="utf-8") as file:
+    METADATA = yaml.safe_load(file)
+
+IMAGEHASHES_PATH = Path(__file__).resolve().parent / "imagehashes.yml"
+with IMAGEHASHES_PATH.open("r", encoding="utf-8") as file:
+    IMAGEHASHES = yaml.safe_load(file)
+HASH_SIZE = 16
+MAX_PHASH_DISTANCE = 2
```

Comment:
it'd be cool if you could add a couple inline comments why choose these particular values

Reply:
Done in https://github.com/ESMValGroup/ESMValTool/pull/4002/commits/192e5c85ca0d86349302e906fb32050b280c90cc